File: components_page/components/card/outline.py
import dash_bootstrap_components as dbc
import dash_html_components as html

card_content = [
    dbc.CardHeader("Card header"),
    dbc.CardBody(
        [
            html.H5("Card title", className="card-title"),
            html.P(
                "This is some card content that we'll reuse",
                className="card-text",
            ),
        ]
    ),
]

row_1 = dbc.Row(
    [
        dbc.Col(dbc.Card(card_content, color="primary", outline=True)),
        dbc.Col(dbc.Card(card_content, color="secondary", outline=True)),
        dbc.Col(dbc.Card(card_content, color="info", outline=True)),
    ],
    className="mb-4",
)

row_2 = dbc.Row(
    [
        dbc.Col(dbc.Card(card_content, color="success", outline=True)),
        dbc.Col(dbc.Card(card_content, color="warning", outline=True)),
        dbc.Col(dbc.Card(card_content, color="danger", outline=True)),
    ],
    className="mb-4",
)

row_3 = dbc.Row(
    [
        dbc.Col(dbc.Card(card_content, color="light", outline=True)),
        dbc.Col(dbc.Card(card_content, color="dark", outline=True)),
    ]
)

cards = html.Div([row_1, row_2, row_3])

File: docs/components_page/components/button/outline.jl
using DashBootstrapComponents, DashHtmlComponents

buttons = html_div([
    dbc_button("Primary", outline = true, color = "primary", className = "me-1"),
    dbc_button("Secondary", outline = true, color = "secondary", className = "me-1"),
    dbc_button("Success", outline = true, color = "success", className = "me-1"),
    dbc_button("Warning", outline = true, color = "warning", className = "me-1"),
    dbc_button("Danger", outline = true, color = "danger", className = "me-1"),
    dbc_button("Info", outline = true, color = "info", className = "me-1"),
    dbc_button("Light", outline = true, color = "light", className = "me-1"),
    dbc_button("Dark", outline = true, color = "dark"),
]);
